docs/bin/plugin_formatter.py

```diff
@@ -516,7 +516,7 @@ def process_plugins(module_map, templates, outputname, output_dir, ansible_versi
         else:
             doc['returndocs'] = None
 
-        doc['author'] = doc.get('author', ['UNKNOWN'])
+        doc['author'] = doc.get('author', ['UNKNOWN']) or []
```

Comment:
I would rewrite this as:
```python
    doc['author'] = doc.get('author') or []
```

I don't see what `['UNKNOWN']` will bring to the table ? If `author` is missing, we expect and empty list too.

Reply:
Since authors aren't removed, there is no need to update this code in this PR :)